src/test/java/org/verdictdb/core/querying/QueryExecutionPlanSimplifierTest.java

```diff
@@ -67,7 +67,7 @@ static void populateData(DbmsConnection conn, String schemaName, String tableNam
         "aggregationBlockColumn", "subsampleColumn", "tierColumn", 3);
   }
 
-  @Test
+  //@Test
```

Comment:
Why are these test cases disabled?

Reply:
I find we are using simplify2() instead of simplify() method when simplifying execution plan. In simplify(), there is a deepcopy step that cannot deepcopy SQL connection object.

Reply:
I see. Can you add the following comment before the class declaration?

"In favor of simplify2() (instead of the legacy function simplify()), we disable these tests."